setup.py

```diff
@@ -351,6 +356,8 @@ def add_dir_name(dir_name, filenames):
     filename = swig_path.lib_filename
     if platform.system() == 'Linux':
```

Comment:
Okay, so let's stick to a single way of determining the OS then, I think that it's good to have a single consistent way